spec/std/thread_spec.cr

```diff
@@ -49,4 +49,17 @@ describe Thread do
 
     thread.join
   end
+
+  it "names the thread" do
+    Thread.current.name.should be_nil
+    name = nil
+
+    thread = Thread.new(name: "some-name") do
+      name = Thread.current.name
+    end
+    thread.name.should eq("some-name")
```

Comment:
Just to be sure:

```suggestion
    thread.name.should eq("some-name")
    name.should be_nil
```

Reply:
There would be a race condition:

- thread is started (immediately)
- main thread is preempted
- thread completes
- main thread resumes
- main thread tries to check `name.should be_nil` :boom: